tests/test_subtyping_processing.py

```diff
@@ -6,6 +6,7 @@
 from bio_hansel.subtype import Subtype
 from bio_hansel.subtyper import absent_downstream_subtypes, sorted_subtype_ints, count_periods, empty_results
 from bio_hansel.utils import find_inconsistent_subtypes
+from bio_hansel.subtype_stats import SubtypeCounts
```

Comment:
Is `SubtypeCounts` used in this file?

Reply:
I believe that was a mistype that I forgot to correct

Reply:
It is not used in this commit but it is used to test the regex in a previous one and it was added here early by mistake so it should be there I guess